flake.nix

```diff
@@ -114,21 +115,17 @@
               pkgs.cabal-fmt
               pkgs.nixpkgs-fmt
               pkgs.fourmolu
-            ];
-            treefmt = pkgs.treefmt;
+              ];
+              treefmt = pkgs.treefmt;
+            };
           };
-        };
 
-        devShells = import ./nix/hydra/shell.nix {
-          inherit inputs pkgs hsPkgs system compiler;
-        };
+          devShells = import ./nix/hydra/shell.nix {
+            inherit inputs pkgs hsPkgs system compiler;
+          };
 
-        # Build selected derivations in CI for caching
-        hydraJobs = pkgs.lib.optionalAttrs (!pkgs.stdenv.isDarwin) {
-          packages = { inherit (packages) hydra-node hydra-tui hydraw spec; };
-          devShells = { inherit (devShells) default; };
         };
```

Comment:
Why can't we keep these?

Reply:
In flake-parts, there must be a configuration option for every attribute in perSystem. hydraJobs doesn't exist in flake-parts because by convention we *should* just build all derivations that can be discovered.

Reply:
Let's kill it and see how it goes.